src/ralph/cli.py

```diff
@@ -560,6 +560,9 @@ def runserver(backend: str, host: str, port: int, **options):
             key = f"RALPH_BACKENDS__DATABASE__{backend_name}__{field_name}".upper()
             if isinstance(value, tuple):
                 value = ",".join(value)
+            if isinstance(value, dict):
+                value = json.dumps(value).replace("'", '"')
+                value = f"'{value}'"
```

Comment:
This string substitution should not have any effect, I think you may need to wrap the serialized JSON string:

```suggestion
            if isinstance(value, dict):
                value = f"'{json.dumps(value)}'"
```

Reply:
That's true!